src/state-incentive-details.tsx

```diff
@@ -89,18 +91,37 @@ const formatTitle = (incentive: Incentive, msg: MsgFn) => {
   }
 };
 
-const formatIncentiveType = (payment_methods: IncentiveType[], msg: MsgFn) =>
-  payment_methods[0] === 'tax_credit'
-    ? msg('Tax credit')
-    : payment_methods[0] === 'pos_rebate'
-    ? msg('Upfront discount')
-    : payment_methods[0] === 'rebate'
-    ? msg('Rebate')
-    : payment_methods[0] === 'account_credit'
-    ? msg('Account credit')
-    : payment_methods[0] === 'performance_rebate'
-    ? msg('Performance rebate')
-    : msg('Incentive');
+const formatIncentiveType = (payment_methods: IncentiveType[]) => {
+  const { msg } = useTranslated();
+  const label =
+    payment_methods[0] === 'tax_credit'
+      ? msg('Tax credit')
+      : payment_methods[0] === 'pos_rebate'
+      ? msg('Upfront discount')
+      : payment_methods[0] === 'rebate'
+      ? msg('Rebate')
+      : payment_methods[0] === 'account_credit'
+      ? msg('Account credit')
+      : payment_methods[0] === 'performance_rebate'
+      ? msg('Performance rebate')
+      : payment_methods[0] === 'assistance_program'
+      ? msg('Assistance program')
+      : msg('Incentive');
+
+  return (
+    <div className="flex gap-1 items-center">
+      {label}
+      <span className="text-purple-500">
+        <QuestionIcon
+          aria-label={msg('Show glossary')}
+          w={10}
+          h={10}
+          opacity={1.0}
```

Comment:
It's because the same QuestionIcon component is used for the tooltip icons in the form, where it has to be 50% opacity. I guess I should make it so 100% is the default, though, and the usage in the form has to specify 50% explicitly. I'll change that.